### PR TITLE
feat: bump reanimated 3

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [PHIRE-125] 

### Description

Update to reanimated 3, removed the worklets and tested out the app, will proceed with merging today possibly.

Linking also:
- original PR: https://github.com/artsy/palette-mobile/pull/125
- revert of original PR due to making it to eigen main before we were ready https://github.com/artsy/palette-mobile/pull/131

[PHIRE-125]: https://artsyproduct.atlassian.net/browse/PHIRE-125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by araujobarret — APPROVED** [review]
Glog is also updated but all good!

**Comment by gkartalis** [comment]
The return of the GLOG lol, will revert the glog

File: ios/Podfile.lock [hunk 1]
```diff
@@ -638,7 +640,7 @@ SPEC CHECKSUMS:
   Flipper-RSocket: d9d9ade67cbecf6ac10730304bf5607266dd2541
   FlipperKit: cbdee19bdd4e7f05472a66ce290f1b729ba3cb86
   fmt: ff9d55029c625d3757ed641535fd4a75fedc7ce9
-  glog: 3d02b25ca00c2d456734d0bcff864cbc62f6ae1a
+  glog: 04b94705f318337d7ead9e6d17c019bd9b1f6b1b
```
Comment (gkartalis) [inline ios/Podfile.lock:643]:
```suggestion
  glog: 3d02b25ca00c2d456734d0bcff864cbc62f6ae1a
```

**Comment by artsyit** [comment]
:rocket: PR was released in [`v11.2.14`](https://github.com/artsy/palette-mobile/releases/tag/v11.2.14) :rocket:

**Pull request merged**